packages/babel-plugin-transform-es2015-arrow-functions/test/fixtures/arrow-functions/spec/expected.js

```diff
@@ -14,7 +14,7 @@ function foo() {
 
     return function () {
       babelHelpers.newArrowCheck(this, _this2);
-      return this;
```

Comment:
Since it is `.bind` both work, this way is just a little more consistent with the non-spec arrow transform, since they share more code now.